src/scores.jl

```diff
@@ -1,7 +1,7 @@
 abstract type AbstractScore end
 
 
-struct DistrictAggregate <: AbstractScore
+mutable struct DistrictAggregate <: AbstractScore
```

Comment:
I decided to go ahead and make all score structs mutable - that's the only way to get partial initialization for `DistrictScore` and `PlanScore`. I also went ahead and changed `DistrictAggregate` and `CompositeScore` to mutable as well for the sake of consistency. It's not a big deal in my mind, since we're already using `deepcopy` in other places whenever the user passes in an array of `Score`s.

Reply:
Wait—couldn't you just change the type of `name` to `Union{String, Missing}` and initialize `name` to `missing` in the one-argument constructor? I feel like just _not defining_ the `name` field could be dangerous and is not semantically useful to users. Doing it this way would also avoid making things `mutable`, which I've heard can have (minor) effects on performance and is IMO a code smell.

Reply:
great point, i agree! best not to make structs mutable, but also explicitly having `name` be missing is more informative

Reply:
great points, folks! changing now 👍